src/ipvs/ip_vs_blklst.c

```diff
@@ -50,12 +50,17 @@ static uint32_t dp_vs_blklst_rnd;
 static inline uint32_t blklst_hashkey(const union inet_addr *vaddr,
                                      const union inet_addr *blklst)
 {
+    /* jhash hurts performance, we do not use it here
+     *
     return rte_jhash_2words((uint32_t) vaddr->in.s_addr, 
```

Comment:
Just remove the code， we can get everything back by git, not need leave unused codes.
but there can be a comment for that.